.github/workflows/release-installer.yaml

```diff
@@ -66,6 +68,34 @@ jobs:
         run: sudo installer -pkg Finch-${GITHUB_REF_NAME}-aarch64.pkg -target /
       - name: Run e2e tests
         run: INSTALLED=true make test-e2e
+      - name: Checkout the tag
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref_name }}
+          fetch-depth: 0
+          persist-credentials: false
+          submodules: true
+      - name: Build project
+        run: |
+          export PATH="/opt/homebrew/opt/libtool/libexec/gnubin:$PATH"
+          which libtool
+          make
+      - name: Multiple instances of Finch test
+        run: |
+          # start two Finch VM
+          ./_output/bin/finch vm init
+          ./_output/bin/finch pull alpine
+          ./_output/bin/finch run -d --name test-ctr1 alpine
+          finch vm init
+          # reinitialize the VMs and verify the container is usable
+          finch vm pull alpine
+          finch run -d --name test-ctr2 alpine
+          ./_output/bin/finch vm stop
+          ./_output/bin/finch vm remove && ./_output/bin/finch vm start 
```

Comment:
won't the `vm start` command fail here because the vm has been removed?

Reply:
You are right, that's a typo. Should be `vm init`.